backstory/src/main/webapp/perspective-demo.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <title>Perspective</title>
+    <link rel="stylesheet" href="style.css" />
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Montserrat" />
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto" />
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Lato" />
+    <script src="script.js"></script>
```

Comment:
Is there a potentially more specific name for `script.js` that would better indicate why it's used here?

Reply:
Usually, I just leave a call to ```script.js``` in case we have any methods to be used throughout the website. However, currently that script statement does not even need to be there (there's no methods used in perspective-demo from ```script.js```, so I can just remove the script call.

Reply:
Done